index.html

```diff
@@ -1,47 +1,264 @@
+          <div class="three">
+            <div class="tags">
+              <span class="canopy-color">Canopy</span>
+              <img class="rectangle" src="images/Counter.png" alt="counter">
+              <span>Back End Dev</span>
+              <img class="rectangle" src="images/Counter.png" alt="counter">
+              <span>2015</span></div>
+          </div>
+          <div class="four">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div class="five">
+            <div class="tags2">
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">html</p>
+                </div>
+              </div>
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">css</p>
+                </div>
+              </div>
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">javaScript</p>
+                </div>
+              </div>
+            </div>
+            <div class="six"><button type="button" class="card-button">See Project</button></div>
+          </div>
+        </div>
+      </div>
 
-      <ul class="social-media">
+      <div class="card">
+        <div class="card-structure">
+          <div class="one"><img src="images/ball.png " class="img-displayer" alt="ball"></div>
+          <div class="two">
+            <h1 class="project-title">Multi-Post Stories</h1>
+          </div>
+          <div class="three">
+            <div class="tags">
+              <span class="canopy-color">Canopy</span>
+              <img class="rectangle" src="images/Counter.png" alt="counter">
+              <span>Back End Dev</span>
+              <img class="rectangle" src="images/Counter.png" alt="counter">
+              <span>2015</span></div>
+          </div>
+          <div class="four">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div class="five">
+            <div class="tags2">
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">html</p>
+                </div>
+              </div>
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">css</p>
+                </div>
+              </div>
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">javaScript</p>
+                </div>
+              </div>
+            </div>
+            <div class="six"><button type="button" class="card-button">See Project</button></div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about">
+      <h1 class="project-title">About</h1>
+      <h1 class="project-title">Myself</h1>
```

Comment:
[OPTIONAL]

- [x] Conventionally you shouldn't have more than one `<h1>` in the same section. (Usually, it is suggested to have only one `<h1>` tag per page)

```suggestion
      <h1 class="project-title">About <br>Myself</h1>
```

Reply:
I took your advise